int8 robot_id
nav_msgs/Odometry position
